udata_front/theme/gouvfr/datagouv-components/src/components/InformationPanel/InformationPanel.vue

```diff
@@ -0,0 +1,190 @@
+                <div class="fr-grid-row fr-grid-row--gutters fr-text--sm fr-m-0">
+                    <div v-for="(value, key) in props.dataset?.extras" :key="key" class="fr-col-12 fr-col-sm-6 fr-col-md-4">
+                        <h3 class="subtitle fr-mb-1v">{{ key }}</h3>
+                        <p class="fr-text--sm fr-m-0 text-mention-grey ">{{ value }}</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </article>
+    <article v-if="props.dataset?.harvest">
+        <header
+            class="fr-grid-row fr-grid-row--middle fr-pb-3w fr-mb-3w border-bottom border-default-grey"
+            :class="{'border-bottom': !harvestExpanded}"
+        >
+            <div class="fr-col">
+                <h2 class="subtitle subtitle--uppercase fr-m-0">{{ $t('Harvest') }}</h2>
+            </div>
+            <div class="fr-col-auto">
+                <button
+                    @click="expand"
+                    role="button"
+                    :aria-expanded="harvestExpanded"
+                    class="fr-btn fr-btn--tertiary-no-outline fr-btn--icon-left fr-btn--secondary-grey-500"
+                    :class="{'fr-icon-arrow-up-s-line': harvestExpanded, 'fr-icon-arrow-down-s-line': !harvestExpanded}"
+                >
+                    <template v-if="harvestExpanded">
+                        {{ $t('Close details') }}
+                    </template>
+                    <template v-else>
+                        {{ $t('See harvest') }}
+                    </template>
+                </button>
+            </div>
+        </header>
+        <div
+            class="accordion-content"
+            ref="harvestRef"
+        >
+            <div class="fr-pb-3w fr-mb-3w border-bottom border-default-grey">
+                <div class="fr-grid-row fr-grid-row--gutters fr-text--sm fr-m-0">
+                    <div v-for="(value, key) in props.dataset?.harvest" :key="key" class="fr-col-12 fr-col-sm-6 fr-col-md-4">
+                        <h3 class="subtitle fr-mb-1v">{{ key }}</h3>
+                        <p class="fr-text--sm fr-m-0 text-mention-grey ">{{ value }}</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </article>
+</template>
+
+<script setup lang="ts">
+import { ref, defineProps, onMounted } from 'vue';
+import { config } from "../../config";
+import { Dataset } from "../../types/datasets";
+import CopyButton from "../CopyButton/CopyButton.vue";
+import { toggleAccordion } from "../../helpers/toggleAccordion";
+import { templateRef } from "@vueuse/core";
+import { formatDate } from '../../helpers/index';
+import { getGranularity, fetchGranularities } from '../../api/granularity';
+import { getFrequencies, fetchFrequencies } from '../../api/frequency';
+
+const props = defineProps<{
+    dataset: Dataset,
+    license: Object
+}>();
+const embedText = ref<string>(
```

Comment:
Can you make a composable like `useEmbed(type, id)` ?

Type should be : "dataset" | "reuse"

This could allow use to use a `URL` instead of a concat for the oembed.js file path